### ISSUE TITLE
AsciiDoc spelling

### ISSUE THREAD

**Issue by noraj** [streetsidesoftware/vscode-spell-checker#1973]
The spell checker is working for AsciiDoc but unlike for Markdown it is triggering _Unknown word_ for words inside image links, inside code blocks, in document attributes tag, etc. so it's checking the spell like if it was bare text and is not aware of the asciidoc syntax.

**Comment by Jason3S** [comment]
@noraj,

A dictionary for AsciiDoc's has not yet been created: [CSpell Dictionaries](https://github.com/streetsidesoftware/cspell-dicts#all-dictionaries), so the spell checker is not aware of the context and treats it like a `plaintext` document. Feel free to open a PR or new Issue there.

Ignoring words starting with an uppercase letter or all upper case words is problematic and leads to many false negatives.

It is possible for you to do this yourself using the configuration:

```jsonc
    "cSpell.languageSettings": [
        {
            "languageId": "asciidoc",
            "ignoreRegExpList": [
                "/\\b[A-Z][a-z]+\\b/g" // Ignore Leading Cap words
            ]
        }
    ],
```

Personally, I think it is better to keep a dictionary of proper names.
